app/screens/home/account/components/options/custom_status/custom_label/custom_label.tsx

```diff
@@ -71,8 +75,8 @@ const CustomLabel = ({customStatus, isCustomStatusExpirySupported, isStatusSet,
                 <FormattedText
                     id={'custom_status.failure_message'}
                     defaultMessage='Failed to update status. Try again'
-                    style={styles.retryMessage}
                     testID='account.custom_status.failure_message'
+                    style={styles.retryMessage}
```

Comment:
Nit: This change is probably not needed.

Reply:
reverted